recipes/configs/gemma/2B_lora.yaml

```diff
@@ -0,0 +1,83 @@
+# Config for multi-device LoRA finetuning in lora_finetune_distributed.py
+# using a gemma 2B model
+#
+# This config assumes that you've run the following command before launching
+# this run:
+#   tune download google/gemma-2b --hf-token <HF_TOKEN> --output-dir /tmp/gemma --ignore-patterns ""
+#
+# To launch on 4 devices, run the following command from root:
+#   tune run --nnodes 1 --nproc_per_node 4 lora_finetune_distributed --config gemma/2B_lora
+#
+# You can add specific overrides through the command line. For example
+# to override the checkpointer directory while launching training
+# you can run:
+#   tune run --nnodes 1 --nproc_per_node 4 lora_finetune_distributed --config gemma/2B_lora checkpointer.checkpoint_dir=<YOUR_CHECKPOINT_DIR>
```

Comment:
I think this was mentioned on Discord, but given that Gemma is such a small model already, any reason to not also add a single-device LoRA config here?

Reply:
Yes, I overlooked that. Now, it support lora and qlora on single device.